toolchain/check/testdata/alias/fail_control_flow.carbon

```diff
@@ -4,12 +4,14 @@
 //
 // AUTOUPDATE
 
-// CHECK:STDERR: fail_control_flow.carbon:[[@LINE+9]]:11: ERROR: Semantics TODO: `Control flow expressions are currently only supported inside functions.`.
+// CHECK:STDERR: fail_control_flow.carbon:[[@LINE+11]]:11: ERROR: Semantics TODO: `Control flow expressions are currently only supported inside functions.`.
```

Comment:
To note on blank lines, we already have non-import-related tests which print multiple diagnostics for the same line of code. I do think that the blank line makes the separation clearer in tests.